src/cli/command/secrets_enable.rs

```diff
@@ -0,0 +1,148 @@
+use clap::{Args, Parser};
+use derive_more::Deref;
+
+use crate::{
+    api::sys::{MountConfigInput, MountInput},
+    cli::{
+        command::{self, CommandExecutor},
+        util::ensure_trailing_slash,
+    },
+    errors::RvError,
+    utils::{
+        kv_builder::KvPairParse,
+        token_util::{DEFAULT_LEASE_TTL, MAX_LEASE_TTL},
+    },
+};
+
+#[derive(Parser, Deref)]
+#[command(
+    author,
+    disable_version_flag = false,
+    about = r#"Enables a secrets engine. By default, secrets engines are enabled at the path
+corresponding to their TYPE, but users can customize the path using the
+-path option.
+
+Once enabled, RustyVault will route all requests which begin with the path to the
+secrets engine.
+
+Enable the AWS secrets engine at aws/:
+
+    $ rvault secrets enable aws
+
+Enable the SSH secrets engine at ssh-prod/:
+
+    $ rvault secrets enable -path=ssh-prod ssh
+
+Enable the database secrets engine with an explicit maximum TTL of 30m:
+
+    $ rvault secrets enable -max-lease-ttl=30m database
+
+For a full list of secrets engines and examples, please see the documentation."#
+)]
+pub struct Enable {
+    #[arg(required = true, next_line_help = false, value_name = "TYPE", long_help = r#"engine type"#)]
+    engine_type: String,
+
+    #[deref]
+    #[command(flatten, next_help_heading = "HTTP Options")]
+    http_options: command::HttpOptions,
+
+    #[command(flatten, next_help_heading = "Command Options")]
+    options: EnableOptions,
+}
+
+#[derive(Args)]
+#[group(required = false, multiple = true)]
+pub struct EnableOptions {
+    #[arg(
+        long,
+        next_line_help = false,
+        value_name = "string",
+        default_value = "",
+        long_help = r#"Place where the secrets engine will be accessible. This must be unique
```

Comment:
`Place where the secrets engine will be accessible`

->

`A place that a secrets engine is enabled at, thus it can be accessed by this path.`

Reply:
fixed